internal/backend/db/dbgorm/vars.go

```diff
@@ -0,0 +1,116 @@
+package dbgorm
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"gorm.io/gorm/clause"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/db/dbgorm/scheme"
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkerrors"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+func (db *gormdb) SetVars(ctx context.Context, vars []sdktypes.Var) error {
```

Comment:
please split all relevant functions to SDK/schema for testability and add `vars_test.go` (see env_tests.go removed lines)

Reply:
https://linear.app/autokitteh/issue/ENG-768/db-vars-testing

Reply:
@itayd merged the tests into your PR